src/Microsoft.ML.Transforms/Text/NgramHashingTransformer.cs

```diff
@@ -866,7 +866,7 @@ public VBuffer<ReadOnlyMemory<char>>[] SlotNamesMetadata(out VectorDataViewType[
     /// |  |  |
     /// | -- | -- |
     /// | Does this estimator need to look at the data to train its parameters? | Yes |
-    /// | Input column data type | Vector of [Key](<xref:Microsoft.ML.Data.KeyDataViewType>) |
+    /// | Input column data type | Vector of [Key](xref:Microsoft.ML.Data.KeyDataViewType) |
```

Comment:
>Key [](start = 46, length = 3)

I would just do <xref:Microsoft.ML.Data.KeyDataViewType>

Reply:
I think it's desirable to keep it as Ivan did, at least I believe that's the convention for key type: name it key, and link the KeyDataViewType.

---
In reply to: [277109991](https://github.com/dotnet/machinelearning/pull/3440#discussion_r277109991) [](ancestors = 277109991)